### PR TITLE
Sprint

### PR DESCRIPTION
**Pull request by csivek**
Testing permissions on pulling into master

### CODE REVIEW

**Review by BigDub — DISMISSED** [review]
Testing "Request changes" option.

File: Source/BeatBoxers/Implementation/BasicFretboardFeed.h [hunk 1]
```diff
@@ -29,14 +29,13 @@ class BEATBOXERS_API UBasicFretboardFeed : public UObject, public IFretboardFeed
...
 
-	void SetDuration(float NewDuration) { NoteDuration = NewDuration; }
-
 	/** Begin IFretboardFeed Implementation */
 	virtual TArray<FFeedNoteData> GetExistingNotes() override;
 	virtual void PauseFeed() override;
 	virtual void ResumeFeed() override;
 	virtual void ClearFeed() override;
 	virtual float GetNoteLifetime() override { return NoteDuration; }
+	virtual void SetNoteLifetime(float NewLifetime) override { NoteDuration = NewLifetime; }
```
Comment (BigDub) [inline Source/BeatBoxers/Implementation/BasicFretboardFeed.h:38]: Testing a comment in a review.

**Review by BigDub — COMMENTED** [review]
Testing "Comment", feedback without explicit approval.

File: Source/BeatBoxers/Implementation/BasicFretboardFeed.h (hunk 1 above)
Comment (csivek) [reply Source/BeatBoxers/Implementation/BasicFretboardFeed.h:38]: Reply

**Review by BigDub — APPROVED** [review]
Testing Approve.

**Pull request merged**